extra/Addons/SetUp/Executor.TCC.farconfig

```diff
@@ -2,6 +2,6 @@
EAT;RESOLUTION;RESTOREPOINT;RETURN;REXEC;RMDIR;RSHELL;SAVECONSOLE;SCREEN;SCREENMONITOR;SCRIPT;SCRPUT;SELECT;SENDHTML;SENDMAIL;SERVICEMONITOR;SERVICES;SET;SETARRAY;SETDOS;SETERROR;SETLOCAL;SETP;SHIFT;SHORTCUT;SHRALIAS;SMPP;SNMP;SNPP;SPONGE;SREPLACE;SSHEXEC;START;STATUSBAR;SWITCH;SYNC;TABCOMPLETE;TAIL;TAR;TASKBAR;TASKDIALOG;TASKEND;TASKLIST;TCDIALOG;TCEDIT;TCFILTER;TCFONT;TCTOOLBAR;TEE;TEXT;THREAD;TIME;TIMER;TITLE;TMP;TOAST;TOUCH;TPIPE;TRANSIENT;TREE;TRUENAME;TS;TYPE;UNALIAS;UNBZIP2;UNFUNCTION;UNGZIP;UNJAR;UNLIBRARY;UNMOUNTISO;UNMOUNTVHD;UNQLITE;UNSET;UNSETARRAY;UNSETP;UNTAR;UNZIP;UPTIME;USBMONITOR;UUID;VBEEP;VDESKTOP;VER;VERIFY;VIEW;VOL;VSCRPUT;WAITFOR;WAKEONLAN;WEBFORM;WEBSOCKET;WEBUPLOAD;WHICH;WINDOW;WINSTATION;WMIQUERY;WMIRUN;WSETTINGS;WSHELL;WSHORTCUT;Y;ZIP;ZIPSFX;*?;*7UNZIP;*7ZIP;*ACTIVATE;*ALIAS;*ASSOC;*ASSOCIATE;*ATTRIB;*BATCOMP;*BDEBUGGER;*BEEP;*BREAK;*BREAKPOINT;*BTMONITOR;*BZIP2;*CALL;*CANCEL;*CAPTURE;*CD;*CDD;*CHCP;*CHDIR;*CHRONIC;*CLIP;*CLIPMONITOR;*CLS;*COLOR;*COMMANDS;*COMMENT;*COPY;*COPYDIR;*DATE;*DATEMONITOR;*DEBUGMONITOR;*DEBUGSTRING;*DEDUPE;*DEFER;*DEL;*DELAY;*DESCRIBE;*DESKTOP;*DETACH;*DIFFER;*DIR;*DIRENV;*DIRHISTORY;*DIRS;*DISKMONITOR;*DNS;*DO;*DRAWBOX;*DRAWHLINE;*DRAWVLINE;*ECHO;*ECHOERR;*ECHOS;*ECHOSERR;*ECHOX;*ECHOXERR;*EJECTMEDIA;*ENDLOCAL;*ENUMPROCESSES;*ENUMSERVERS;*ENUMSHARES;*ERASE;*ESET;*EVENTLOG;*EVENTMONITOR;*EVERYTHING;*EXCEPT;*EXIT;*EXPR;*FFIND;*FILELOCK;*FIREWIREMONITOR;*FOLDERMONITOR;*FONT;*FOR;*FREE;*FTYPE;*FUNCTION;*GLOBAL;*GOSUB;*GOTO;*GZIP;*HASH;*HEAD;*HELP;*HISTORY;*IDE;*IF;*IFF;*IFTP;*INKEY;*INPUT;*INSTALLED;*JABBER;*JAR;*JOBMONITOR;*JOBS;*JOINDOMAIN;*JUMPLIST;*KEYBD;*KEYS;*KEYSTACK;*LIBRARY;*LINKS;*LIST;*LOADBTM;*LOADMEDIA;*LOCAL;*LOCKMONITOR;*LOG;*LOOKUPERRORS;*LUA;*MD;*MEMORY;*MKDIR;*MKLINK;*MKLNK;*MONITOR;*MOUNTISO;*MOUNTVHD;*MOVE;*MOVEDIR;*MSGBOX;*NETMONITOR;*NOTIFY;*ODBC;*ON;*OPTION;*OSD;*PATH;*PAUSE;*PDIR;*PEE;*PIPEVIEW;*PLAYAVI;*PLAYSOUND;*PLUGIN;*POPD;*POSTMSG;*POWERMONITOR;*PRINT;*PRINTF;*PRIORITY;*PROCESSMONITOR;*PROMPT;*PSHELL;*PSUBST;*PUSHD;*QUERYBOX;*QUIT;*RD;*REBOOT;*RECORDER;*RECYCLE;*REGDIR;*REGMONITOR;*REM;*REN;*RENAME;*REPEAT;*RESOLUTION;*RESTOREPOINT;*RETURN;*REXEC;*RMDIR;*RSHELL;*SAVECONSOLE;*SCREEN;*SCREENMONITOR;*SCRIPT;*SCRPUT;*SELECT;*SENDHTML;*SENDMAIL;*SERVICEMONITOR;*SERVICES;*SET;*SETARRAY;*SETDOS;*SETERROR;*SETLOCAL;*SETP;*SHIFT;*SHORTCUT;*SHRALIAS;*SMPP;*SNMP;*SNPP;*SPONGE;*SREPLACE;*SSHEXEC;*START;*STATUSBAR;*SWITCH;*SYNC;*TABCOMPLETE;*TAIL;*TAR;*TASKBAR;*TASKDIALOG;*TASKEND;*TASKLIST;*TCDIALOG;*TCEDIT;*TCFILTER;*TCFONT;*TCTOOLBAR;*TEE;*TEXT;*THREAD;*TIME;*TIMER;*TITLE;*TMP;*TOAST;*TOUCH;*TPIPE;*TRANSIENT;*TREE;*TRUENAME;*TS;*TYPE;*UNALIAS;*UNBZIP2;*UNFUNCTION;*UNGZIP;*UNJAR;*UNLIBRARY;*UNMOUNTISO;*UNMOUNTVHD;*UNQLITE;*UNSET;*UNSETARRAY;*UNSETP;*UNTAR;*UNZIP;*UPTIME;*USBMONITOR;*UUID;*VBEEP;*VDESKTOP;*VER;*VERIFY;*VIEW;*VOL;*VSCRPUT;*WAITFOR;*WAKEONLAN;*WEBFORM;*WEBSOCKET;*WEBUPLOAD;*WHICH;*WINDOW;*WINSTATION;*WMIQUERY;*WMIRUN;*WSETTINGS;*WSHELL;*WSHORTCUT;*Y;*ZIP;*ZIPSFX" />
```

Comment:
Just to clarify: the whole point of this key is to instruct Far that certain strings are internal commands of the active interpreter that do not exist in the file system as separate executables and therefore should not be even searched there.

Are you sure that things like 7UNZIP, 7ZIP, PSHELL etc. are internal commands?

Reply:
Correct. These are all internal commands. You don't need to install 7-Zip or have `7z.exe` available in order to use these commands. TCC has over 265 [internal commands](https://jpsoft.com/help/index.htm?tcc_commands.htm), including a dozen of them for compression/decompression.

### Internal Commands including 7zip

![tcc-7zip-commands](https://github.com/FarGroup/FarManager/assets/800999/9fe1726e-f987-45cf-a11a-a05f1f153c03)

### 7zip Built-in User Interface

![tcc-7zip-dialog](https://github.com/FarGroup/FarManager/assets/800999/7fbaaf62-cae2-480f-9646-62f43c8b1aac)

### 7zip Command Help

![tcc-7zip-help-file](https://github.com/FarGroup/FarManager/assets/800999/206c24b9-0e67-45e0-b063-edc2622857dd)

Reply:
Impressive. Thank you, merged.